src/main/controllers/service/ChooseServiceAreaController.ts

```diff
@@ -110,17 +111,26 @@ export class ChooseServiceAreaController {
     return false;
   }
 
-  private async tryGetServiceData(
+  /**
+   *
+   * @param serviceChosen the chosen service
+   * @param action action
+   * @param serviceAreasPageData current service area page data
+   * @param hasErrors errors
+   * @param lng language
+   * @param res required so it can error out
+   */
+  private async safeGetServiceData(
     serviceChosen: string,
     action: string,
     serviceAreasPageData: ServiceAreasData,
     hasErrors: boolean,
     lng: string,
     res: Response
-  ) {
+  ): Promise<ServiceAreasData | null> {
```

Comment:
If we remove line 135 (return null) then we may not need to have the | null here too